src/main/resources/templates/users/notifications.ftlh

```diff
@@ -156,7 +155,7 @@
 
                             <div class="invite-message invite-accepted" style="display: none;">
                                 <i class="minor fas fa-3x fa-birthday-cake"></i><br/>
-                                @Html(messages("notification.invite.joined", subject.name))<br/>
+                                @Html(<@spring.messageArgs "notification.invite.joined"  subject.name />)<br/>
```

Comment:
you can remove the outer html wrap in these cases

Reply:
Yes, freemarker doesn't escape HTML, unless you tell it to